src/unix/stream.c

```diff
@@ -837,6 +841,30 @@ static void uv__write(uv_stream_t* stream) {
 #else
     while (n == -1 && errno == EINTR);
 #endif
+#ifdef MSG_FASTOPEN
+  } else if (stream->flags & UV_TCP_FASTOPEN) {
+    struct sockaddr *addr = { 0 };
```

Comment:
make this a `struct sockaddr_storage`, `struct sockaddr` is not large enough for an IPv6 address. Also allocate it on the stack.